webcompat/templates/home-page/issue-wizard-form.html

```diff
@@ -362,5 +364,74 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
       </div>
     </div>
   </div>
-  {{ form.hidden_tag() }}
+    {{ form.hidden_tag() }}
 </form>
+
+<div class="popup-overlay popup-modal__close"></div>
+
+<div
+  class="popup-modal shadow"
+  data-popup-modal="what-is-webcompat">
+  <img class="close popup-modal__close" src="../../img/svg/icons/svg-cross-black.svg" title="Close popup" alt="Close popup" />
```

Comment:
This `src` definition will be broken in the actual deployment. Please use the same static imports like we do in the rest of the template eg:
```
templates/home-page/issue-wizard-form.html:    <img class="warning-icon" src="{{ url_for('static', filename='img/svg/icons/svg-info-icon.svg') }}" title="Warning" alt="Warning" />

```

Reply:
This also has other occurences